### PR TITLE
Creating video-list.json file for Ethical Hacking 

### PR DESCRIPTION
**Pull request by mrpatil007**
Creating video-list.json file and folder for Ethical Hacking Video's

### CODE REVIEW

File: video-categories/Ethical-Hacking/video-list.json [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+      "video_author": null
+    },
+    {
+      "title": "Ethical Hacking: Proxy & Packet Filtering",
+      "complete": false,
+      "script_author": null,
+      "video_author": null
+    },
+    {
+      "title": "Ethical Hacking:  Denial of Service (Dos and DDos Attacks)",
```
Comment (atjonathan) [inline video-categories/Ethical-Hacking/video-list.json]:
```diff
- "title": "Ethical Hacking:  Denial of Service (Dos and DDos Attacks)",
+ "title": "Ethical Hacking: Denial of Service (Dos and DDos Attacks)",
```

File: video-categories/Ethical-Hacking/video-list.json [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+      "complete": false,
+      "script_author": null,
+      "video_author": null
+    },
+    {
+      "title": "Ethical Hacking:  Social Engineering",
```
Comment (atjonathan) [inline video-categories/Ethical-Hacking/video-list.json]:
```diff
- "title": "Ethical Hacking:  Social Engineering",
+ "title": "Ethical Hacking: Social Engineering",
```

File: video-categories/Ethical-Hacking/video-list.json [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+      "complete": false,
+      "script_author": null,
+      "video_author": null
+    },
+    {
+      "title": "Ethical Hacking:  Stenography",
```
Comment (atjonathan) [inline video-categories/Ethical-Hacking/video-list.json]:
```diff
- "title": "Ethical Hacking:  Stenography",
+ "title": "Ethical Hacking: Stenography",
```

File: video-categories/Ethical-Hacking/video-list.json [hunk 4]
```diff
@@ -0,0 +1,72 @@
...
+      "complete": false,
+      "script_author": null,
+      "video_author": null
+    },
+    {
+      "title": "Ethical Hacking:  Cryptography",
```
Comment (atjonathan) [inline video-categories/Ethical-Hacking/video-list.json]:
```diff
- "title": "Ethical Hacking:  Cryptography",
+ "title": "Ethical Hacking: Cryptography",
```

Comment (mrpatil007) [inline video-categories/Ethical-Hacking/video-list.json]: oh i forget to remove space

**Comment by atjonathan** [comment]
@mrpatil007, please change the folder to be lowercase.
Using `git mv Ethical-Hacking ethical-hacking`

**Comment by atjonathan** [comment]
@mrpatil007 please change the folder to be lowercase using `git mv Ethical-Hacking ethical-hacking`.

**Comment by mrpatil007** [comment]
@atjonathan ok sure

**Comment by mrpatil007** [comment]
@atjonathan  i rename it

**Comment by atjonathan** [comment]
### **LGTM :+1:**
Thanks @mrpatil007 :shipit:

**Pull request merged**